app/config/FrontendAppConfig.scala

```diff
@@ -109,6 +110,13 @@ class FrontendAppConfig @Inject() (configuration: Configuration, phase5Switch: P
       s"$departureFrontendUrl/local-reference-number"
     }
 
+  val declareArrivalStartWithMRNUrl: String =
```

Comment:
There is a method above which already does this

  val declareArrivalNotificationStartUrl: String =
    if (phase5Switch.Arrivals.enabled) {
      declareArrivalNotificationUrlBase
    } else {
      s"$declareArrivalNotificationUrlBase/movement-reference-number"
    }